project-base/app/src/Component/Router/FriendlyUrl/FriendlyUrlMatcher.php

```diff
@@ -49,6 +49,7 @@ public function match($pathinfo, RouteCollection $routeCollection, DomainConfig
             $matchedParameters['_controller'] = 'Symfony\Bundle\FrameworkBundle\Controller\RedirectController::urlRedirectAction';
```

Comment:
the $matchedParameters array is created dynamically if I'm not mistaken... Is it ok?